src/include/rbd/librbd.h

```diff
@@ -577,6 +577,14 @@ CEPH_RBD_API int rbd_mirror_mode_get(rados_ioctx_t io_ctx,
                                      rbd_mirror_mode_t *mirror_mode);
 CEPH_RBD_API int rbd_mirror_mode_set(rados_ioctx_t io_ctx,
                                      rbd_mirror_mode_t mirror_mode);
+CEPH_RBD_API int rbd_mirror_remote_namespace_get(rados_ioctx_t io_ctx,
+                                                 char *remote_namespace,
+				                 size_t *max_len);
+/* The value can be set only if the mirror mode is disabled. The previously set
+ * value will be automatically removed when the pool or namespace is disabled.
+ */
```

Comment:
The "pool or namespace" bit is a bit vague and technically incorrect because disabling mirroring on the containing pool wouldn't reset the value on a namespace.  Also, it would be nice to clarify what happens when the value is "removed". Perhaps something like:
```suggestion
/**
 * The value can be set only if mirroring on io_ctx is disabled. The previously
 * set value will be automatically reset to io_ctx's namespace when mirroring on
 * io_ctx is disabled.
 */
```

Reply:
The pool or namespace was meant to refer to what the io_ctx was pointing to.